packages/msw-addon/src/mswDecorator.js

```diff
@@ -1,6 +1,12 @@
 let api
 
-const IS_BROWSER = typeof global.process === 'undefined'
+const IS_BROWSER = () => {
+  try {
+    return this === window;
```

Comment:
`window` and `process` are two different ways to check for the browser environment. The `window` option is less reliable, as the window object can be polyfilled by non-browser environments (i.e. JSDOM), which should still be treated as _Node.js_. 

Would it work if we explicitly added the check for `global`?

```js
const IS_BROWSER = global && typeof global.process === 'undefined'
```

Reply:
You are still referencing `global` like it exists, but it may not, resulting in an error. Hence the `try/catch`

Reply:
We can check whether `global` exists via `typeof global !== 'undefined'` before accessing it. There's no need for try/catch here.

Reply:
Try it in your browser console.